src/main/resources/db/migration/0350/V357__AskTerraware.sql

```diff
@@ -0,0 +1,36 @@
+CREATE EXTENSION IF NOT EXISTS vector;
+CREATE EXTENSION IF NOT EXISTS hstore;
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE vector_store (
+    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+    content TEXT,
+    metadata JSONB,
+    embedding vector(1536)
+);
```

Comment:
I know we're not wanting to spend a ton of time on this experimental feature. Just curious if the name `vector_store` is too generic

Reply:
It's somewhat hardwired into Spring AI's data storage classes, sadly.